pyproject.toml

```
@@ -0,0 +1,47 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pynvml"
+version = "12.0.0"
+description = "Python utilities for the NVIDIA Management Library"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { file = "LICENSE.txt" }
```

It can be useful to include [the SPDX value]( https://spdx.org/licenses/ ) here

```suggestion
license = { text = "BSD-3-Clause" }
```

Can package the license file separately using `tool.setuptools` field `license-files` as illustrated in a separate comment in this review